docs/examples/viz_fine_tuning_gl_context.py

```diff
@@ -0,0 +1,160 @@
+"""
+=======================================================
+Fine-tuning the OpenGL state using shader callbacks
+=======================================================
+
+VTK it’s powerfully, but sometimes we need to get more control about how
```

Comment:
Remove from "VTK to specific actor".

Switch to: 
This example shows how to change the OpenGL state of one or more actors.

Reply:
Remove VTK it's powerful. Just say:
Sometimes we need to get more control on how OpenGL with render the actors.

Reply:
Done